recipes/default.rb

```diff
@@ -15,3 +15,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+user 'seagl'
+
+include_recipe 'base::sudo'
+
+sudo 'seagl' do
+  user 'seagl'
+  runas 'root'
+  nopasswd true
+end
```

Comment:
Take a look at `proj-misc::apereo` and see how we use `users_manage` instead and how we also directly call the sudo recipe instead of our wrapper.